docs/guides/qiskit-addons-mpf-get-started.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<p>You could emphasize the relation between using the 2nd-order Suzuki formula here and having chosen <code>order=2</code> earlier on. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>